tests/config/pytest_software.yml

```diff
@@ -0,0 +1,712 @@
+abacas:
```

Comment:
This file is now called `pytest_modules.yml`. Please remove this file and update teh `pytest_modules` accordingly.

Reply:
I will update this.